tests/mocha/block_test.js

```diff
@@ -1294,37 +1306,23 @@ suite('Blocks', function () {
         });
         test('Set While Visible - Editable', function () {
           this.block.setCommentText('test1');
-          const icon = this.block.getCommentIcon();
-          icon.setVisible(true);
+          const icon = this.block.getIcon(Blockly.icons.CommentIcon.TYPE);
```

Comment:
why not use the `getCommentIcon` method? makes the test more readable imo, you can have a separate test for that method if you want to test it